bigframes/session.py

```diff
@@ -321,7 +323,10 @@ def __init__(
             ),
         )
 
-        self._bq_connection = context.bq_connection
+        self._bq_connection = (
+            context.bq_connection
```

Comment:
We should transform this internally to format <project>.<location>.<connection-id> if user just provides <connection-id>. Ashley added this handling in remote function, may be we could generalize that: https://github.com/googleapis/python-bigquery-dataframes/blob/c0efec8956198247b27904345a795f09c80d3502/bigframes/remote_function.py#L767

Reply:
We can make it happen, but right now remote function session is an optional field. Which makes supplementing connection project and location from session not always possible. Do we want to make remote function always take bpd.get_default_session() as back up?

Reply:
I think that's a good idea. Current remote_function session logic pre-dates the default session. we can use default session as a backup if session is not provided

Reply:
Synced offline, we can add bigframes default session to remote function, so remote function will always have a session, and add a helper to share the functionality of supplementing connections.


def helper(default_proj:str, default_loc:str, default_conn_id:str) -> str: